packages/lens/vite.config.js

```diff
@@ -0,0 +1,19 @@
+/** @type {import('vite').UserConfig} */
+export default {
+  build: {
+    rollupOptions: {
+      external: [/@rabbitholegg/],
```

Comment:
do I need to add anything here for external dependancies (ie: apollo)?

Reply:
I'm curious about this too @sammccord